src/main/kotlin/mate/academy/model/IsoscelesTrapezoid.kt

```diff
@@ -0,0 +1,16 @@
+package mate.academy.model
+
+class IsoscelesTrapezoid(
+        color: Color,
+        private val firstBase: Int,
+        private val secondBase: Int,
+        private val height: Int
+) : Figure(color) {
+
+    override fun getArea(): String {
+        return "Figure: ${this::class.simpleName}, area: ${((firstBase + secondBase) / 2.0) * height} sq. units, " +
```

Comment:
You may use triple quote marks with related extension function to avoid concatenation

Reply:
done